doc/src/modules/physics/mechanics/symsystem.rst

```diff
@@ -0,0 +1,184 @@
+motion that simulation code can be designed around.
+
+SymbolicSystem Example Usage
+============================
+
+This code will go over the manual input of the equations of motion for the
+simple pendulum into `SymbolicSystem` using x and y coordinates instead of
+theta.
+
+The equations of motion are formed at
+http://nbviewer.jupyter.org/github/bmcage/odes/blob/master/docs/ipython/Planar%20Pendulum%20as%20DAE.ipynb ::
+
+    >>> from sympy import atan, symbols, Matrix
+    >>> from sympy.physics.mechanics import (dynamicsymbols, ReferenceFrame,
+    ...                                      Particle, Point)
+    >>> import sympy.physics.mechanics.system as system
+
+The first step will be to initialize all of the dynamic and constant symbols. ::
+
+    >>> x, y, u, v, lam = dynamicsymbols('x y u v lambda')
+    >>> m, l, g = symbols('m l g')
+
+Next step is to define the equations of motion in multiple forms:
+
+    [1] Explicit form where the kinematics and dynamics are combined
+        x' = F(x, t, r, p)
+
+    [2] Implicit form where the kinematics and dynamics are combined
+        M(x, p) x' = F(x, t, r, p)
+
+    [3] Implicit form where the kinematics and dynamics are separate
+        M(q, p) u' = F(q, u, t, r, p)
+        q' = G(q, u, t, r, p)
+
+where
+
+    x : states, i.e. [q, u]
+    t : time
+    r : specified (exogenous) inputs
+    p : constants
+    q : generalized coordinates
+    u : generalized speeds
+    M : mass matrix or generalized inertia matrix (implicit dynamical
+        equations or combined dynamics and kinematics)
+    F : right hand side (implicit dynamical equations or combined dynamics
+        and kinematics)
+    G : right hand side of the kinematical differential equations ::
+
+    >>> dyn_implicit_mat = Matrix([[1, 0, -x/m],
+    ...                            [0, 1, -y/m],
+    ...                            [0, 0, l**2/m]])
+    >>> dyn_implicit_rhs = Matrix([0, 0, u**2 + v**2 - g*y])
+    >>> comb_implicit_mat = Matrix([[1, 0, 0, 0, 0],
+    ...                             [0, 1, 0, 0, 0],
+    ...                             [0, 0, 1, 0, -x/m],
+    ...                             [0, 0, 0, 1, -y/m],
+    ...                             [0, 0, 0, 0, l**2/m]])
+    >>> comb_implicit_rhs = Matrix([u, v, 0, 0, u**2 + v**2 - g*y])
+    >>> kin_explicit_rhs = Matrix([u, v])
+    >>> comb_explicit_rhs = comb_implicit_mat.LUsolve(comb_implicit_rhs)
+
+Now the reference frames, points and particles will be set up so this
+information can be passed into `system.SymbolicSystem` in the form of a bodies
+and loads iterable. ::
+
+    >>> theta = atan(x/y)
+    >>> N = ReferenceFrame('N')
+    >>> A = N.orientnew('A', 'Axis', [theta, N.z])
+    >>> O = Point('O')
+    >>> P = O.locatenew('P', l * A.x)
+    >>> Pa = Particle('Pa', P, m)
+
+Now the bodies and loads iterables need to be initialized. ::
+
+    >>> bodies = [Pa]
+    >>> loads = [(P, g * m * N.x)]
```

Comment:
The idea of the example is to showcase coming up with the equations by hand and entering them manually into `SymbolicSystem`. `LagrangesMethod` should eventually have a method `to_system` and so I don't know that it would be ideal to have the example on `SymbolicSystem`'s page to manually transfer data from `LagrangesMethod`.